core-bundle/contao/dca/tl_user.php

```diff
@@ -571,7 +571,7 @@ public function switchUser($row, $href, $label, $title, $icon)
 		$router = System::getContainer()->get('router');
 		$url = $router->generate('contao_backend', array('_switch_user'=>$row['username']));
 
-		return '<a href="' . $url . '" title="' . StringUtil::specialchars($title) . '">' . Image::getHtml($icon, $label) . '</a> ';
+		return '<a href="' . $url . '" title="' . StringUtil::specialchars($title) . '" data-turbo-prefetch="false">' . Image::getHtml($icon, $label) . '</a> ';
```

Comment:
We have to do the same in the `tl_member` DCA, otherwise hovering over the "log in as user" button would change the front end session.